src/platform/testing/e2e/cypress/support/form-tester/index.js

```diff
@@ -228,7 +228,7 @@ const defaultPostHook = pathname => {
   return () => {
     cy.findByText(/continue/i, {
       selector: 'button',
-      timeout: 10000,
+      timeout: 20000,
```

Comment:
Without this change Cypress tests were failing in CI.

Reply:
That might be good to know for future CI failures.